src/app/pages/home/home.component.html

```diff
@@ -2,7 +2,7 @@
 <div class="home-container">
   <div class="main-section">
     <mat-card class="column">
-      <img mat-card-image src="/assets/icons/icon-square.svg">
+      <img width="1" height="1" mat-card-image src="/assets/icons/icon-square.svg">
```

Comment:
Do you think it would be all right if you added a comment explaining what the `width` and `height` attributes do?

(I'm having trouble figuring out what's going on here. Googling suggests that this might be a trick used to get an image into the browser cache extra-quickly? Am I getting that right?)

Thanks a bunch! 🙂

Reply:
I'm doing it to hint to the browser what the aspect ratio of the image is (1x1 is arbitrary, any x/y=1 would work) to prevent layout shift when the image loads.

Reply:
I'm not sure I want to add a comment about it into the HTML since HTML comments don't get stripped away when the app is compiled.

Reply:
Ah, I see, that makes sense! Cool!

Hmm… I can see why you might feel that way about HTML comments. What about a comment in `home.component.scss` instead? It's a bit out of the way, but we are adjusting the size of the mat-card-image there, so that might be a good place to mention it :)

Reply:
Great idea. Done!